docs/discover/search.asciidoc

```diff
@@ -66,7 +79,6 @@ To load a saved search into Discover:
 If the saved search is associated with a different index pattern than is currently
 selected, opening the saved search also changes the selected index pattern.
```

Comment:
Great point, it'll automatically select the language that the saved search was written in. Added a note.